contracts/interfaces/paymasters/IBiconomyTokenPaymaster.sol

```diff
@@ -32,6 +32,8 @@ interface IBiconomyTokenPaymaster {
      */
     event FeeReceiverChanged(address indexed _oldfeeReceiver, address indexed _newfeeReceiver, address indexed _actor);
 
+    event TokensWithdrawn(address indexed _token, address indexed _to, uint256 indexed _amount, address actor);
```

Comment:
Not sure `_amount` should be `indexed`

Reply:
we can index 3 params and I think it's fine. also out of scope of remediations PR